src/datatypes/colorizer.md

```diff
@@ -3,9 +3,11 @@
 A colorizer specifies a mapping between values and pixels/objects of an output image.
 Different variants of colorizers perform different kinds of mapping.
 
-Usually, there are two miscellaneous fields in each colorizer, namely `noDataColor` and `defaultColor`.
+Usually, there are three miscellaneous fields in each colorizer, namely `noDataColor`, `overColor` and `underColor`.
```

Comment:
It is not that easy anymore.

It should be something like this.

There are colors for values or value ranges, then a color for NO DATA, and then color(s) for data that is out of range, e.g. over and under colors for gradients or a default color for palettes.

Reply:
61fc7724b85e21f45ba69bd81c464b7010e4a8ac

Ok, I've rewritten it much more deliberately now.